### PR TITLE
feat: alert avatar documentation

### PR DESCRIPTION
**Pull request by pankaj-prog**

### CODE REVIEW

**Comment by Ramandeep9898** [comment]
docu. part is super clean
rather than navbar make a side bar for components because in future u won't be having just 8-9 components .
although ui is clean u can work on landing page

**Comment by gauravsinhaweb** [comment]
Looks clean and elegant. Good to merge!

File: docs.html [hunk 1]
```diff
@@ -10,10 +10,26 @@
...
+          <a href="#">
+            <i class="fab fa-github"></i>
+          </a>
+          <a href="#">
+            <i class="fab fa-linkedin"></i>
+          </a>
+          <a href="#">
+            <i class="fab fa-twitter"></i>
+          </a>
+        </div>
```
Comment (manojsarna) [inline docs.html:24]: this is good!!

File: style.css [hunk 2]
```diff
@@ -12,41 +13,99 @@
...
+  margin-left: auto;
+}
+
+.site-header-ctas i {
+  min-width: 1.2em;
+  aspect-ratio: 1;
+  display: flex;
+  place-content: center;
+  border-radius: 50%;
+  padding: 6px;
```
Comment (manojsarna) [inline style.css:88]: good class naming :)

File: style.css [hunk 3]
```diff
@@ -84,3 +190,45 @@ h2 ~ .badge-text-container {
...
+.token.string,
+.token.variable {
+  color: rgb(1, 67, 97);
+}
+
+/* TEXT CONTENT  */
+code[class*="language-"],
+pre[class*="language-"] {
+  color: var(--SUCCESS);
+}
```
Comment (manojsarna) [inline style.css:228]: using comments in the code is great 👍.

**Review by manojsarna — APPROVED** [review]
overall code is good.

Nice and simple UI!!

File: docs.html (hunk 1 above)
Comment (theVivek-kumar) [reply docs.html:24]: this is looking good I like the naming convention

File: src/components/alert/alert.html [hunk 4]
```diff
@@ -31,10 +48,88 @@
...
+          <span class="highlighted">alert</span> along with the class for color.
+        </p>
+        <section class="component-example-wrapper-secondary">
+          <div class="alert alert-error">
+            <i class="fas fa-exclamation-circle alert-icon"></i>
+            <p class="alert-message">This is an error alert — check it out!</p>
+          </div>
+          <div class="alert alert-warning">
+            <i class="fas fa-exclamation-triangle alert-icon"></i>
+            <p class="alert-message">This is a warning alert — check it out!</p>
```
Comment (theVivek-kumar) [inline src/components/alert/alert.html:80]: this looks good to me

File: src/components/avatar/avatar.html [hunk 5]
```diff
@@ -30,12 +46,152 @@
...
+                    class="responsive-img"
+                    src="...."
+                    alt="...."
+                  />
+              </div>
+              <!-- Medium Squared Avatar -->
+              <div class="avatar avatar-md avatar-square">
+                  <img
+                    class="responsive-img"
+                    src="...."
```
Comment (theVivek-kumar) [inline src/components/avatar/avatar.html:186]: spacing and everything looks nice

**Review by theVivek-kumar — APPROVED** [review]
looks good to me !!

**Pull request merged**